docs/contributing/css_guidelines.rst

```diff
@@ -3,35 +3,6 @@ CSS coding guidelines
 
 Our CSS is written in `Sass <https://sass-lang.com/>`_, using the SCSS syntax.
 
-Compiling
-~~~~~~~~~
```

Comment:
I have removed this because those are essentially the same instructions as other front-end build tools now.